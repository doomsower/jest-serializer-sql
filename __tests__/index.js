const knex = require('knex');

const sql = `SELECT id, foo, bar, 'test_type' AS type FROM table1 UNION ALL
select id, title FROM table2 ORDER BY id DESC, title ASC NULLS LAST`;

test('should beautify SQL', () => {
  const pg = knex({ client: 'pg'});

  expect(pg.raw(sql)).toMatchSnapshot();
});
