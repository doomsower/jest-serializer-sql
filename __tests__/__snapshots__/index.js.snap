// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should beautify SQL 1`] = `
SELECT
    id,
    foo,
    bar,
    'test_type' AS type
FROM
    table1
UNION ALL
select
    id,
    title
FROM
    table2
ORDER BY
    id DESC,
    title ASC NULLS LAST

`;
